Modules/posixmodule.c

```diff
@@ -21,6 +21,7 @@
       FSCTL_GET_REPARSE_POINT is not exported with WIN32_LEAN_AND_MEAN. */
 #  include <windows.h>
 #  include <pathcch.h>
+#define EX_OK EXIT_SUCCESS
```

Comment:
```suggestion
#endif

#if !defined(EX_OK) && defined(EXIT_SUCCESS)
#define EX_OK EXIT_SUCCESS
```

Let's define this by its actual condition, rather than making it a platform check.